jenkins/build.sh

```diff
@@ -60,9 +60,9 @@ rpm -q rpm-build > /dev/null || {
 # Create version file
 echo $VER > "$BASE_DIR"/VERSION
 /bin/chmod +rx "$BASE_DIR"/VERSION
-/bin/chmod +x "$BASE_DIR"/src/setup/rgw_setup
-/bin/chmod +x "$BASE_DIR"/src/setup/rgw_service
-#/bin/chmod +x "$BASE_DIR"/src/support/rgw_support_bundle
+/bin/chmod +x "$BASE_DIR"/src/rgw/setup/rgw_setup
+/bin/chmod +x "$BASE_DIR"/src/rgw/setup/rgw_service
+#/bin/chmod +x "$BASE_DIR"/src/rgw/support/rgw_support_bundle
```

Comment:
do we need this code.. if not required then we can remove this

Reply:
@rahul-tripathi-git  second option seems to be feasible. 
This code will highlight all files have executable permission so in future if we want to add more files (or change existing file permissions) then this will be easy to follow same pattern and it will ensure that we will have uniform set of permission for all these files.